Networking/Networking/Requests/RESTRequest.swift

```diff
@@ -58,38 +55,7 @@ struct RESTRequest: URLRequestConvertible {
 }
 
 extension RESTRequest {
```

Comment:
nit: do you think we can add `RESTRequest` to the [documentation on `URLRequestConvertible`](https://github.com/woocommerce/woocommerce-ios/blob/148b0a955a8390c5d3218bfd8e7c19de709c6c07/docs/NETWORKING.md#urlrequestconvertible)?

Reply:
Update docs in f1511f824cb5fd9c5746d807f70ccca6bfeb531e and e30bfd550c4f612e268335e043dd356ea436153a